tests/integration/targets/inventory_terraform_state_azurerm/aliases

```diff
@@ -0,0 +1 @@
+cloud/azure
```

Comment:
These Azure integration tests [still aren't running in CI](https://github.com/ansible-collections/cloud.terraform/actions/runs/7729206444/job/21071842535#step:11:327). I'm not sure if that's expected behavior? If so I think we should add `unsupported` here and update the github actions workflow. If not, we should investigate.

Reply:
They are not running because this PR is updating the integration workflow with the Azure credentials generation

Reply:
@hakbailey the tests are running here as you can see on this testing PR
https://github.com/ansible-collections/cloud.terraform/pull/115